extensionsGallery.json

```diff
@@ -5080,7 +5080,68 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+				{
+				"extensionId": "100",
+				"extensionName": "ads-searcheverywhere",
+				"displayName": "Search Everywhere",
+				"shortDescription": "Search tables, views, stored procedures, functions by name or code.",
+				"publisher": {
+					"displayName": "MikhailT",
+					"publisherId": "MikhailT",
+					"publisherName": "MikhailT"
+				},
+				"versions": [
+					{
+						"version": "1.0.0",
+						"lastUpdated": "07/23/2023",
+						"assetUri": "",
+						"fallbackAssetUri": "fallbackAssetUri",
+						"files": [
+							{
```

Comment:
as a third party extension, please use download page instead of vsix package, please refer to this example: https://github.com/MikhailProfile/azuredatastudio/blob/patch-1/extensionsGallery.json#L267

Reply:
Updated